kmir/k-src/mir-types.md

```diff
@@ -526,14 +524,13 @@ Hooked functions
 ```k
 module MIR-HOOKS
   imports STRING
-  imports MIR-SYNTAX
+  imports MIR-TYPE-SYNTAX
 ```
 
 We use several hooks which convert between token and string representations:
 
 ```k
-  syntax String ::= StringLiteral2Sring(StringLiteral) [function, total, hook(STRING.token2string)]
-  syntax StringLiteral ::= String2SringLiteral(String) [function, total, hook(STRING.string2token)]
+  syntax String ::= StringLiteral2String(StringLiteral) [function, total, hook(STRING.token2string)]
```

Comment:
Why was `StringLiteral2Sring` removed?

Reply:
This was removed because I didn't see it being used; I suppose it is a bit out of scope.